### DISCUSSION TITLE
Latest boilerplate?

### DISCUSSION THREAD

**Discussion by hyperknot** [alex8088/electron-vite#624] [Q&A] [answered]
Hi,

What is the latest boilerplate? Is it
https://github.com/alex8088/electron-vite-boilerplate
or
https://github.com/alex8088/quick-start/tree/master/packages/create-electron

I see they are similar, but not the same.

**Comment by alex8088** [comment, marked as answer]
Recommend: `npm create @quick-start/electron@latest`

**Reply by hyperknot** [reply]
Thanks!
